main.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/smallfish/simpleyaml"
+)
+
+/*
+	I am sure there's a better way to do this, I'm just trying to get the hang of Go right now.
+	Edvin probably has something better for CLI stuff already.
+*/
+func main() {
```

Comment:
Hmm I'm ok with merging this as a first pass at a cli, but think we should use one of these two projects for the cli we go with in the end

https://github.com/spf13/cobra
https://github.com/urfave/cli

Reply:
I'm messing around with cobra on my branch right now.

Reply:
Edvin got the goodies. It was more of a learning experience for me with Go. I actually kinda like Go now and I think Edvin might be able to use my YAML parser to some extent. We can and will make it better. 😺

Reply:
I did indeed use the yaml parser ^ @theinfamousrj